projects/google.com/gcloud/package.yml

```diff
@@ -0,0 +1,49 @@
+versions:
+  url: https://cloud.google.com/sdk/docs/install-sdk
+  match: /google-cloud-cli-\d+\.\d+\.\d+-darwin-arm\.tar\.gz/
+  strip:
+    - /^google-cloud-cli-/
+    - /-darwin-arm\.tar\.gz$/
+dependencies:
+  python.org: ^3.11
+
+build:
+  dependencies:
+    curl.se: '*'
+  script:
+    - mkdir -p {{prefix}}/libexec
+    - curl -L $DISTRIBUTABLE_URL | tar -xz --strip-components=1
+    - ./install.sh $ARGS
```

Comment:
should this be `warnings: vendored` or does `install.sh` build something?

Reply:
> should this be `warnings: vendored` or does `install.sh` build something?

you're right, we can definitely add `warnings: vendored` here